crates/bevy_sprite/src/mesh2d/material.rs

```diff
@@ -416,20 +447,30 @@ pub fn queue_material2d_meshes<M: Material2d>(
                 // -z in front of the camera, the largest distance is -far with values increasing toward the
                 // camera. As such we can just use mesh_z as the distance
                 sort_key: FloatOrd(mesh_z),
-                // This material is not batched
-                batch_size: 1,
+                // Batching is done in batch_and_prepare_render_phase
+                batch_range: 0..1,
+                dynamic_offset: None,
             });
         }
     }
 }
 
+#[derive(Component, Clone, Default, PartialEq, Eq, Deref, DerefMut)]
+pub struct Material2dBindGroupId(Option<BindGroupId>);
```

Comment:
This could be Copy, which I think is preferable. Could use `copied` instead of `cloned` in GetBatchData.

Reply:
Same for the 3d type

Reply:
Done.